src/MicrosoftTeams.ts

```diff
@@ -485,11 +488,11 @@ namespace microsoftTeams {
   let initializeCalled = false;
 
   let isFramelessWindow = false;
-  let currentWindow: Window;
-  let parentWindow: Window;
+  let currentWindow: any | any;
+  let parentWindow: Window | any;
   let parentOrigin: string;
   let parentMessageQueue: MessageRequest[] = [];
-  let childWindow: Window;
+  let childWindow: Window | any;
```

Comment:
Why do you need to change the type here (and above)? In what scenario does this need to be `any`?

Reply:
Looks like we are adding new Window interface that override WIndow interface. will fix it